docs/maps/api-ui.md

```diff
@@ -86,4 +86,24 @@ WA.ui.registerMenuCommand("test", () => {
 
 <div class="col">
     <img src="https://workadventu.re/img/docs/menu-command.png" class="figure-img img-fluid rounded" alt="" />
-</div>
\ No newline at end of file
+</div>
+
+
+
+### Awaiting User Confirmation (with space bar)
+
+```typescript
+triggerMessage(message: string): TriggerMessage
```

Comment:
According to the code, "triggerMessage" accepts a callback as the second argument. We should document that.